planet_scope/ndwi/eob.js

```diff
@@ -0,0 +1,50 @@
+//VERSION=3
+//PlanetScope NDWI EO Browser
+
+function setup() {
+    return {
+        input: [{
+            bands: [
+                "green",
+                "nir",
+                "dataMask",
+                "clear"]
+        }],
+        output: [
+            { id: "default", bands: 4 },
+            { id: "index", bands: 1, sampleType: "FLOAT32" },
+            { id: "eobrowserStats", bands: 2, sampleType: "FLOAT32" },
+            { id: "dataMask", bands: 1 },
+        ]
+    }
+}
+
+function evaluatePixel(sample) {
+
+    let band1 = sample.green;
+    let band2 = sample.nir;
+    const denominator = band1 + band2;
+    let ndwi = denominator === 0 ? NaN : (band1 - band2) / denominator;
+
+    const clear = (sample.dataMask * sample.clear);
```

Comment:
```suggestion
    let ndwi = index(sample.green, sample.nir);

    const clear = sample.dataMask && sample.clear;
```

Reply:
thanks! I did not know this, will change all to use index function